### PR TITLE
Cleanup this garbage

### PR DESCRIPTION
**Pull request by Iapetus-11**
Exactly what it sounds like. Minor-ish changes that shouldn't break anything but improve your code. Don't do what you did last time and ignore my pr for a week until it's outdated.

### CODE REVIEW

File: statcord/client.py [hunk 1]
```diff
@@ -16,13 +16,14 @@
 class Client:
     """Client for using the statcord API"""
 
-    def __init__(self, bot, token, **kwargs):
+    def __init__(self, bot, token, *, mem=True, cpu=True, bandwidth=True, debug=False, custom1=None, custom2=None):
         self.logger = logging.getLogger("statcord")
         self.logging_level = kwargs.get("logging_level", logging.WARNING)
```
Comment (anishanne) [inline statcord/client.py]: "Kwargs" is not defined and causes the library to crash.

**Review by anishanne — CHANGES_REQUESTED** [review]

File: statcord/client.py [hunk 2]
```diff
@@ -32,21 +32,21 @@ def __init__(self, bot, token, *, mem=True, cpu=True, bandwidth=True, debug=Fals
...
+            raise TypeError(f"Bandwidth config: expected type bool not {self.bandwidth.__class__.__qualname__}")
 
-        self.debug: bool = debug
-        if not isinstance(debug, bool):
-            raise TypeError(f"Debug config: expected type bool not {debug.__class__.__qualname__}")
+        self.debug: bool = kwargs.get("debug", False)
+        if not isinstance(self.debug, bool):
+            raise TypeError(f"Debug config: expected type bool not {self.debug.__class__.__qualname__}")
 
         self.custom1: Optional[Coroutine] = custom1
```
Comment (anishanne) [inline statcord/client.py]: Custom1 + Custom2 are undefined.

**Review by anishanne — CHANGES_REQUESTED** [review]

File: statcord/client.py [hunk 3]
```diff
@@ -22,10 +22,14 @@ def __init__(self, bot, token, **kwargs):
         self.logger.setLevel(self.logging_level)
 
         if not isinstance(bot, DiscordClient):
-            raise TypeError(f"Expected class deriving from discord.Client for arg bot not {bot.__class__.__qualname__}")
+            raise TypeError(
```
Comment (anishanne) [inline statcord/client.py:25]: This formatting is unnecessary.

**Review by anishanne — CHANGES_REQUESTED** [review]

**Comment by tbnritzdoge** [comment]
i like garbage 😃

**Pull request closed without merging**